pkg/oauth2/active.go

```diff
@@ -62,11 +62,22 @@ func activateCmd(vc *cmdutils.VerbCmd) {
 			"The path to the private key file")
 		set.StringVar(&c.Scope, "scope", c.Scope,
 			"The OAuth 2.0 scope(s) to request")
+		set.StringVar(
```

Comment:
I don't suggest you change this file, their looks confused between `--auth-params` and `oauth2`.